### PR TITLE
remove dependency on kubectl

### PR DESCRIPTION
**Pull request by a7i**
one less dependency we have to worry about

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/approve
thanks @a7i, looks like there's just an unused variable

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -1325,7 +1325,12 @@ func computeCPUMemFraction(t *testing.T, cs clientset.Interface, node *v1.Node,
 	}
 	for _, pod := range allpods.Items {
 		if pod.Spec.NodeName == node.Name {
-			req, _ := utils.PodRequestsAndLimits(&pod)
+			req, lim := utils.PodRequestsAndLimits(&pod)
```
Comment (damemi) [inline test/e2e/e2e_test.go:1328]: this is why ci is failing:
```
# sigs.k8s.io/descheduler/test/e2e [sigs.k8s.io/descheduler/test/e2e.test]
test/e2e/e2e_test.go:1328:9: lim declared but not used
```

**Comment by damemi** [comment]
Looks like we need a 1.19 bump similar to https://github.com/kubernetes-sigs/descheduler/pull/801
working on a pr

**Comment by a7i** [comment]
> Looks like we need a 1.19 bump similar to #801 working on a pr

This is actually done in the 0.25 PR but I guess it makes sense to keep the two separate?

**Comment by damemi** [comment]
Ah okay, I missed that it was done there. It can be easier to keep them separate (or at least in separate commits) but this is fine

**Pull request merged**